lib/has3d.js

```diff
@@ -14,6 +14,12 @@ module.exports = function has3d() {
         };
 
     // Add it to the body to get the computed style.
+    var body;
+    if(!document.body)
```

Comment:
Are you trying to render it server side or loading the scripts in the head? If yes, the best solution would be to just add a check in the first line to also return false if document.body is not available:

```javascript
if (!window.getComputedStyle || !document.body) {
        return false;
} 
````
Could you make this change?